warehouse/oidc/forms/activestate.py

```diff
@@ -12,31 +12,100 @@
 
 import re
 
-from typing import TypedDict
+from collections.abc import Callable
+from typing import Any, TypedDict
 
 import requests
 import sentry_sdk
 import wtforms
 
 from warehouse import forms
 from warehouse.i18n import localize as _
-from warehouse.utils.project import PROJECT_NAME_RE
+from warehouse.oidc.forms._core import PendingPublisherMixin
 
-_VALID_ORG_URL_NAME = re.compile(r"^[a-zA-Z0-9-]{3,40}$")
+_VALID_ORG_URL_NAME_AND_ACTOR_NAME = re.compile(r"^[a-zA-Z0-9-]{3,40}$")
 _VALID_PROJECT_NAME = re.compile(r"^[.a-zA-Z0-9-]{3,40}$")
+_DOUBLE_DADHES = re.compile(r"--+")
```

Comment:
Typo?

```suggestion
_DOUBLE_DASHES = re.compile(r"--+")
```

Reply:
Doh...Indeed. Fixed locally to update where it's used as well.

Reply:
Pushed.